junction/proposals/models.py

```diff
@@ -230,6 +230,11 @@ def get_votes_count(self):
         down_vote_count = ProposalCommentVote.objects.filter(proposal_comment=self, up_vote=False).count()
         return up_vote_count - down_vote_count
 
+    def get_reviewer_nick(self):
+        reviewer = ConferenceProposalReviewer.objects.get(conference_id=self.proposal.conference_id,
+                                                          reviewer_id=self.commenter_id)
```

Comment:
@kracekumar A Question. I am aware `.get` might throw an DoesNotExist error. I didn't bother to capture it because I don't see a place where the commenter is not a reviewer, and yet this method might get called. Please let me know if I have missed anything. Or if I need to capture this exception anyways.